### PR TITLE
fix: changed employmentType type

### PR DESCRIPTION
**Pull request by wawrzonnn**

### CODE REVIEW

**Review by mmcode9407 — APPROVED** [review]
Brilliant

File: src/components/MyProfile/ProfileMain/ProfileMain.tsx [hunk 1]
```diff
@@ -90,10 +92,7 @@ const ProfileMain = async () => {
               {profile?.seniority} {profile?.position} Developer
             </span>
             <div className={styles.addInfo}>
-              <div className={styles.addInfoItem}>
-                {' '}
-                {profile?.employmentType}
-              </div>
+            <div className={styles.addInfoItem}>{mapEmploymentType(profile?.employmentType)}</div>
```
Comment (Sumick) [inline src/components/MyProfile/ProfileMain/ProfileMain.tsx:95]: **nitpick:** Profile should not be optional, we should redirect to homepage if profile does not exist, or redirect to `create-profile` if user is authorized

File: src/utils/mapEmploymentType.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { EmploymentType } from '@prisma/client'
+
+export const mapEmploymentType = (employmentType?: EmploymentType) => {
```
Comment (Sumick) [inline src/utils/mapEmploymentType.ts:3]: **nitpick:** We shouldkeep it `data/profile/mappers.ts`

**Review by Sumick — CHANGES_REQUESTED** [review]
Two important fixes:)

File: src/components/UserProfile/UserProfileMain/UserProfileMain.tsx [hunk 3]
```diff
@@ -49,7 +50,7 @@ const UserProfileMain = ({ userProfile }: { userProfile: ProfileModel }) => {
             </div>
             <div className={styles.addInfo}>
               <div className={styles.addInfoItem}>
-                {userProfile?.employmentType}
+                {mapEmploymentType(userProfile?.employmentType)}
```
Comment (Sumick) [inline src/components/UserProfile/UserProfileMain/UserProfileMain.tsx:53]: **nitpick:** should not be optional

File: src/data/frontend/profile/mappers.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { EmploymentType } from '@prisma/client'
+
+export const mapEmploymentType = (employmentType?: EmploymentType) => {
```
Comment (Sumick) [inline src/data/frontend/profile/mappers.ts:3]: **nitpick:**should not be optional

**Review by Sumick — APPROVED** [review]
Looks good - one more little thing, please fix it before merging:)

**Pull request merged**